packages/edit-site/src/components/dataviews/context.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+const DataViewsContext = createContext( {} );
```

Comment:
What's the idea of this context, can you expand on it a bit? Is it something internal so we avoid prop drilling or something?

Reply:
For this PR yes, but I had it in the other PR for extenders. We can remove it and pass the table instance down where is needed. Do you think I should update it in this PR?

Reply:
I think it's fine but yeah I don't think we should expose that context.

Reply:
I removed it.